scripts/open_failed.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+
+from docopt import docopt
+import logging
+import re
+import os
+import sys
+
+doc = """
+usage: 
+	teuthology-open-failed -h --help
```

Comment:
I would suggest, to rename it to `teuthology-open` and have options like `--failed`, `--dead`, `--passed`, `--running`, etc.
Also provide an option for switching the open tools, overriding environment like TEUTHOLOGY_OPEN, defaults to something like `vim -p`, because someone might want to use `less` instead, or even gui editors, if teuthology is setup on developers machine.

Reply:
we could use the EDITOR environment variable like other tools (e.g. git)